src/token_contract/src/test.nr

```diff
@@ -0,0 +1,14 @@
+mod burn_private;
+mod burn_public;
+mod mint_to_private;
+mod mint_to_public;
+mod mint_to_commitment;
+// mod reading_constants;
```

Comment:
i think this one is already testable, also, it's called `view` in the nft, let's make the names match 🙏